app/uk/gov/hmrc/statepension/domain/nps/NpsSummary.scala

```diff
@@ -96,9 +96,12 @@ case class NpsAmountA2016(
                            post88COD: BigDecimal = 0,
                            graduatedRetirementBenefit: BigDecimal = 0
                          ) {
-   val additionalStatePension: BigDecimal =(pre97AP - (pre88GMP + post88GMP + pre88COD + post88COD)).max(0) + post97AP + post02AP
+   val basicStatePension = basicPension
```

Comment:
Again you may as well parse this as basicStatePension rather than have two class members doing the same thing.